content_scripts/storageLoader.js

```diff
@@ -18,6 +22,48 @@ function addDarkMode() {
     document.getElementsByTagName("head")[0].appendChild(link);
 }
 
+const setMonochrome = () => {
+    var link = document.createElement("link");
+    link.id = "MonochromeCss";
+    link.href = "https://ghcdn.rawgit.org/norbit8/MoodleBooster/yanir-enhance_page/enhance-page/monochrome.css"; // TODO: Change on merge to the main path
+    link.type = "text/css";
+    link.rel = "stylesheet";
+    document.getElementsByTagName("head")[0].appendChild(link);
+}
+
+
+const setFontSize = (sizeValue) => {
+    switch(sizeValue){
+        case "1":
+            document.getElementsByTagName("body")[0].style = "font-size:30px;";
+            return;
+        case "2":
+            document.getElementsByTagName("body")[0].style = "font-size:50px;";
+            return;
+        default:
+            document.getElementsByTagName("body")[0].style = "";
+    }
+}
+
+const setContrast = (contrastValue) => {
+    switch(contrastValue){
```

Comment:
Switch case could be avoided by using a constant array with the relevant numbers which will result in a much neater code (IMO)